cmd/discovery5.go

```diff
@@ -103,5 +128,15 @@ func RunDiscv5(ctx *cli.Context) error {
 	// Discovery5 service
 	// check: https://github.com/migalabs/armiarma/blob/ca3d2f6adea364fc7f38bdabda912b5541bb4154/src/discovery/dv5/dv5_service.go#L58
 	// Bootnodes are in pkg/config/bootnodes
```

Comment:
Coming from the previous [comment](https://github.com/migalabs/eth-light-crawler/pull/1/commits/8db8a34c3974439e6c107152a416d2993b25c686#r1045785021) , you could directly ref the Bootnodes from `pkg/config/bootnodes`

Reply:
How do I call the Bootnodes in `pkg/config/bootnodes`?

Reply:
You can refer to the `config` package in the import section on top of it 
```go
import(
     ...
     "github.com/migalabs/eth-light-crawler/pkg/config"
     ...
)


...
bootnodes := config.Bootnodes 
...
``